scripts/populate_placeholders.py

```diff
@@ -19,6 +19,7 @@ def main():
       "[[HYPERLINK_A012]]": "[A012 - Maintain referral letter](https://developer.nhs.uk/apis/e-Referrals/explore_endpoint_a012.html)",
       "[[HYPERLINK_A015]]": "[A015 - Retrieve appointment slots](#api-Default-a015-retrieve-appointment-slots)",
       "[[HYPERLINK_A016]]": "[A016 - Book or defer appointment](https://developer.nhs.uk/apis/e-Referrals/explore_endpoint_a016.html)",
+      "[[HYPERLINK_A019]]": "[A019 - Generate patient letter](https://developer.nhs.uk/apis/e-Referrals/explore_endpoint_a019.html)",
```

Comment:
Also update the link for A005 to point to : '#api-Default-a005-retrieve-referral-request'. The url path is build from '#api-TAG-OPERATION-ID'

Reply:
Udpated